### PR TITLE
LPS-168318 Replace objectDefinitionId1 and objectDefinitionId2 with objectDefinitionExternalReferenceCode1 and objectDefinitionExternalReferenceCode2

### PR DESCRIPTION
**Pull request by seltong**
Related ticket: https://issues.liferay.com/browse/LPS-168318

### CODE REVIEW

File: modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectRelationshipResourceImpl.java [hunk 1]
```diff
@@ -128,31 +128,23 @@ public ObjectRelationship postObjectDefinitionObjectRelationship(
...
+
+		if (objectDefinition == null) {
+			objectDefinition =
+				_objectDefinitionLocalService.addObjectDefinition(
+					objectRelationship.
+						getObjectDefinitionExternalReferenceCode2(),
+					contextUser.getUserId());
 		}
 
+		long objectDefinitionId2 = objectDefinition.getObjectDefinitionId();
```
Comment (MarinhoFeliphe) [inline modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectRelationshipResourceImpl.java:146]: @seltong inline

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
ci:test:relevant

**Comment by p-albuquerque** [comment]
Resent in: #1378

**Pull request closed without merging**